package.json

```diff
@@ -1250,9 +1250,7 @@
                                             "description": "%vscode-docker.tasks.docker-run.dockerRun.volumes.permissions%",
                                             "enum": [
                                                 "rw",
-                                                "ro",
-                                                "rw,z",
```

Comment:
the ",z" variants were for SE-Linux or something. Any particular reason for removing them?

Reply:
Great question. @danegsta opted to use the newer `--mount` syntax for volume mapping, which does not support the SELinux relabel feature. Docker Inc's guidance on the topic was basically "We don't really want to be in the business of relabeling things, so _you_ do that on your own, before we get involved".

Reply:
Roger